en/Alert.md

```diff
@@ -0,0 +1,22 @@
+### Alert
+
+General guidelines for alerts and trips:
+* If a TripDescriptor is specified in an alert, the informed_entities must provide a `trip_id`. 

```

Comment:
"must" should be changed to "should", or this needs to be moved to the spec.

Reply:
Addressed in https://github.com/MobilityData/GTFS_Realtime_Best-Practices/commit/26dd2eb173316bcf3caaaf93a9a0c9481103bc42.